app.py

```diff
@@ -255,6 +255,7 @@ def create_app() -> Flask:  # noqa: C901
     csrf.exempt(fund_store_bp)
     csrf.exempt(application_store_bp)
     csrf.exempt(assessment_store_bp)
+    csrf.exempt(eligibility_bp)
```

Comment:
Does this need to be exempt? If it's serving HTML and/or used by browsers, it should definitely not be - only the internal machine-to-machine 'API' endpoints should be